src/Character/Controller/BaseController.lua

```diff
@@ -272,14 +271,15 @@ function BaseController:UpdateCharacter(): ()
         local LowerTorsoCFrame = HumanoidRootPartCFrame * self.Character.Attachments.HumanoidRootPart.RootRigAttachment.CFrame * self.Character.Motors.Root.Transform * self.Character.Attachments.LowerTorso.RootRigAttachment.CFrame:Inverse()
         local UpperTorsoCFrame = LowerTorsoCFrame * self.Character.Attachments.LowerTorso.WaistRigAttachment.CFrame * self.Character.Motors.Waist.Transform * self.Character.Attachments.UpperTorso.WaistRigAttachment.CFrame:Inverse()
         local HeadCFrame = UpperTorsoCFrame * self.Character.Attachments.UpperTorso.NeckRigAttachment.CFrame * self.Character.Motors.Neck.Transform * self.Character.Attachments.Head.NeckRigAttachment.CFrame:Inverse()
-        CameraService:UpdateCamera(HeadCFrame * self.Character.Head:GetEyesOffset())
-    else
-        --Update the camera based on the last CFrame if the motors can't update (not in Workspace).
-        local CurrentCameraCFrame = Workspace.CurrentCamera.CFrame
+		CameraService:UpdateCamera(HeadCFrame * self.Character.Head:GetEyesOffset())
+		self.LastCameraCFrame = Workspace.CurrentCamera.CFrame
```

Comment:
Because of `HeadLocked` being `true` in some cases, I am not sure if reading `Workspace.CurrentCamera.CFrame` is safe to do.